pkg/operator/ceph/object/rgw.go

```diff
@@ -123,6 +123,15 @@ func (c *clusterConfig) startRGWPods() error {
 			return fmt.Errorf("failed to create rgw keyring. %+v", err)
 		}
 
+		// Set radosgw config flag before it starts
+		// This works since the keyring has been created
+		if c.clusterInfo.CephVersion.IsAtLeastMimic() {
```

Comment:
Are you adding the mimic checks because of the external clusters supporting luminous? Even with that, our minimum version for deploying these daemons should still be mimic.